<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>321</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url('/home/kkyu/amr_ws/DL/project_deep/face_communication/socket/background.png') 0 stretch;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>231</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 100);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 210);
font: bold 20pt &quot;Ubuntu&quot;;
</string>
   </property>
   <property name="text">
    <string>Hyun ZZOOM</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);
border:none;
border-bottom: 2px solid rgba(105, 118, 132, 255);
color: rgba(255, 255, 255, 230);
padding-bottom: 7px;</string>
   </property>
   <property name="placeholderText">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="labelIP">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);
border:none;
border-bottom: 2px solid rgba(105, 118, 132, 255);
color: rgba(255, 255, 255, 230);
padding-bottom: 7px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 0, 0);
color: rgb(255, 255, 255);
font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
